client/ayon_substancepainter/plugins/publish/collect_textureset_images.py

```diff
@@ -155,6 +155,14 @@ def create_image_instance(self, instance, template, outputs,
         image_instance.data["families"] = [product_type, "textures"]
         if instance.data["creator_attributes"].get("review"):
             image_instance.data["families"].append("review")
+            image_instance.data["fps"] = instance.context.data.get("fps", 25.0)
```

Comment:
I'm still not confident about the fallback to context fps. I wonder if getting the fps from the task entity (with fallback to folder entity if no task is set) of the instance may be the better fallback?